### PR TITLE
Fix ContinueTask user prompt for locales with uppercase yes

### LINKED ISSUE #1717: Answering any Question isnt possible anymore -> everytime abort
Hello,

since few Releases, there is a Problem where Answering a Question from yay isnt possible (expect to enter nothing, only press enter).

For exception:

Import PGP? Y/n > y > abort
Import PGP? Y/n > n > abort
Import PGP? Y/n > *just hit enter* > imports

or

Edit PKGBUILD? Y/n > y > abort
Edit PKGBUILD? Y/n > n > abort
Import PGP? Y/n > *just hit enter* > continues installation 

and so on. 

Every Question, only enter is working. Else yay aborts. Except at the Point Pacman gets started to install packages, there everything works.

### PR DESCRIPTION
**Pull request by Drumstickx**
This pull request fixes the issue outlined in #1667. May also fix #1717

### CODE REVIEW

**Comment by Drumstickx** [comment]
Also fixes #1737

**Comment by Jguer** [comment]
well spotted, that seems to be the issue indeed, thanks, I'll probably make a release soon, just unsure between making a patch only for this or a major including the new sort algorithm

File: pkg/text/text.go [hunk 1]
```diff
@@ -76,5 +76,5 @@ func ContinueTask(s string, cont, noConfirm bool) bool {
 
 	response = strings.ToLower(response)
```
Comment (Jguer) [inline pkg/text/text.go]: In this case you should not need the ToLower if you're using `strings.EqualFold`, at least from checking the documentation

**Review by Jguer — CHANGES_REQUESTED** [review]

File: pkg/text/text.go (hunk 1 above)
Comment (Drumstickx) [reply pkg/text/text.go]: Correct, I missed that when fixing the linter violation. Thanks for the hint.

**Review by Jguer — APPROVED** [review]

**Pull request merged**